docs/shared/features/enhance-AI.md

```diff
@@ -24,14 +24,34 @@ Nx maintains comprehensive metadata about your workspace to power features like
 
 This enhanced context allows your LLM to move beyond simple file-level operations to understand your workspace at an architectural level, making it a more effective development partner.
 
-To **enable LLM enhancement in your workspace**:
+## Setting Up LLM Enhancements
+
+### VS Code with GitHub Copilot
+
+To enable LLM enhancement in VS Code:
 
 1. Install or update [Nx Console](/getting-started/editor-setup) in VS Code
 2. Ensure you have GitHub Copilot installed and configured
 3. Start using Copilot in your Nx workspace by typing `@nx` at the beginning of your prompt - this will automatically provide the enhanced context
```

Comment:
Can we maybe include a screenshot here showing the `@nx` prompt in the chat window? like this one

<img width="608" alt="Screenshot 2025-02-28 at 10 22 07" src="https://github.com/user-attachments/assets/554dce1b-3771-4bad-8bc7-e9cda762849d" />